catalyst_voices/packages/catalyst_voices_repositories/test/src/catalyst_data_gateway_repository/catalyst_data_gateway_repository_test.dart

```diff
@@ -1,82 +1,136 @@
-      final result = await mock.getHealthStarted();
+      final result = await repository.getHealthStarted();
       expect(result.isSuccess, true);
     });
     test('getHealthStarted Internal Server Error', () async {
-      when(mock.getHealthStarted()).thenAnswer((_) async {
-        return Failure(NetworkErrors.internalServerError);
-      });
-
-      final result = await mock.getHealthStarted();
-
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 500), null),
+      );
+      final result = await repository.getHealthStarted();
       expect(result.isFailure, true);
       expect(result.failure, equals(NetworkErrors.internalServerError));
     });
     test('getHealthStarted Service Unavailable', () async {
-      when(mock.getHealthStarted()).thenAnswer((_) async {
-        return Failure(NetworkErrors.serviceUnavailable);
-      });
-      final result = await mock.getHealthStarted();
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 503), null),
+      );
+      final result = await repository.getHealthStarted();
       expect(result.isFailure, true);
       expect(result.failure, equals(NetworkErrors.serviceUnavailable));
     });
 
     test('getHealthReady success', () async {
-      when(mock.getHealthReady()).thenAnswer((_) async => Success(null));
-      final result = await mock.getHealthReady();
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 204), null),
+      );
+      final result = await repository.getHealthReady();
       expect(result.isSuccess, true);
     });
+
     test('getHealthReady Internal Server Error', () async {
-      when(mock.getHealthReady()).thenAnswer((_) async {
-        return Failure(NetworkErrors.internalServerError);
-      });
-      final result = await mock.getHealthReady();
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 500), null),
+      );
+      final result = await repository.getHealthReady();
       expect(result.isFailure, true);
       expect(result.failure, equals(NetworkErrors.internalServerError));
     });
     test('getHealthReady Service Unavailable', () async {
-      when(mock.getHealthReady()).thenAnswer((_) async {
-        return Failure(NetworkErrors.serviceUnavailable);
-      });
-      final result = await mock.getHealthReady();
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 503), null),
+      );
+      final result = await repository.getHealthReady();
       expect(result.isFailure, true);
       expect(result.failure, equals(NetworkErrors.serviceUnavailable));
     });
 
     test('getHealthLive success', () async {
-      when(mock.getHealthLive()).thenAnswer((_) async => Success(null));
-      final result = await mock.getHealthLive();
+      final repository = setupRepository(
+        chopper.Response(http.Response('', 200), null),
```

Comment:
Would `HttpStatus.ok` status work here? It gives a meaningful name to a status code. Wdyt? There will be more places to replace this if you agree:)